packages/tests/components/SfBadge/SfBadge.cy.tsx

```diff
@@ -78,31 +75,28 @@ describe('SfBadge', () => {
     const maxValue = Math.floor(Math.random() * 20);
     it(`should render max value: ${maxValue}`, () => {
       initializeComponent({ max: maxValue, content: 40 });
-
       page().hasContent(maxValue).makeSnapshot();
     });
   });
 
   describe('when prop content is set to string', () => {
     it(`should render the passed string content `, () => {
       initializeComponent({ content: 'test' });
-
```

Comment:
Can you please revet the files to how they were originally - this makes it hard to review this PR 😅